CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
   [Samuel Giddins](https://github.com/segiddins), [Eric Amorde](https://github.com/amorde)
   [#767](https://github.com/CocoaPods/Xcodeproj/pull/767)
 
+* Bump Xcode version to suppress warnings to update build settings  
```

Comment:
@chuganzy can you please rebase and fix this entry? We have since then shipped 1.19.0 version of Xcodeproj.

Reply:
@dnkoutso Done!